df/plugin.go

```diff
@@ -84,6 +84,12 @@ var (
 		"securityfs",
 		"devpts",
 		"mqueue",
+		"hugetlbfs",
+		"nsfs",
```

Comment:
@obourdon, thanks for the details. It's a good idea to have configurable options in the future. Thanks for your work and effort to make Snap framework better!